client/src/components/DataFiles/DataFilesModals/DataFilesSelectModal.jsx

```diff
@@ -91,6 +100,7 @@ const DataFilesSelectModal = ({ isOpen, toggle, onSelect }) => {
                 />
               )}
             </div>
+            <DataFilesShowPathModal api={apiValue} />
```

Comment:
Rather than define a new prop and initiate the dispatch here, can we move it to `DataFilesShowPathModal`? If not, then we can instead use a selector to grab the api value in the `DataFilesShowPathModal` component